test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "mocha": true
+  }
+}
```

Comment:
> Do you prefer enable mocha env in whole project ? Or define env mocha with inline comment /*eslint-env mocha */ in test/index.js

I'm not really sure, but I don't really like this `test/.eslintrc` file too. I would opt to just define it in the main `.eslintrc` as it's not that big of a deal. WDYT @hemanth @SBoudrias?